src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/ApplicationLogic.cs

```diff
@@ -26,5 +26,12 @@ public class ApplicationLogic
         /// </summary>
         [JsonProperty(PropertyName = "schemaRef")]
         public string SchemaRef { get; set; }
+
+        /// <summary>
+        /// Specifies wether anonymous access is allowed in stateless mode or not for this particular data type.
+        /// Defaults to false if not specified.
+        /// </summary>
+        [JsonProperty(PropertyName = "allowAnonymousOnStateless")]
+        public bool AllowAnonymousOnStateless { get; set; } = false;
```

Comment:
Should this be an optional parameter? `bool?`? Otherwise this will be a breaking change where app developers have to make changes to their data types even though anonymous is not allowed by default 🤔

Reply:
Will it be breaking when we set the default to false?

Reply:
@SandGrainOne any thoughts on this? Do we need it to be nullable? It would be good to avoid nullable if we can, my idea was that the default value is equal to the current behavior.

Reply:
Hmm yes, maybe that covers it 🤔

Reply:
JSON deserialization should be able to handle a JSON document without the property. It can easily be tested if unsure.

Reply:
default value of a boolean is false so the assignment is unnecessary, but being explicit can make the intent more clear.

Reply:
Yep, that was the intention. I can write a small test just to make sure.

Reply:
Added tests to verify that it doesn't have to be nullable.